src/controllers/trust.interrupt.controller.ts

```diff
@@ -24,12 +25,12 @@ type TrustDetailPageProperties = {
 const getPageProperties = (
   req: Request,
 ): TrustDetailPageProperties => {
-  const trustId = req.params[config.ROUTE_PARAM_TRUST_ID];
 
   const appData = getApplicationData(req.session);
+  const trustId = `${(appData[TrustKey] ?? []).length + 1}`;
```

Comment:
Would we ever need the trustId for this page? If this page is at the very start of the trust journey then no trust data will be displayed or recorded in the session.  Same goes for the references to `trustData` and `mapCommonTrustDataToPage`. I don't think we'll ever have it here. 
Correct me if I'm wrong